test/e2e/components/vl-infotext.js

```diff
@@ -1,14 +1,20 @@
 const { VlElement } = require('vl-ui-core').Test;
 const { By } = require('selenium-webdriver');
 
-class VlInfotext extends VlElement {  
-
+class VlInfotext extends VlElement {
     async getValue() {
-        return (await this.driver.findElement(By.css('div[data-vl-value]'))).getText();
+        const value = await this.findElement(By.css('div[data-vl-value]'));
+        return value.getText();
     }
 
     async getText() {
-        return (await this.driver.findElement(By.css('div[data-vl-text]'))).getText();
+        const text = await this.findElement(By.css('div[data-vl-text]'));
+        return text.getText();
+    }
```

Comment:
text ook uit meer dan text alleen bestaan, dus hier ook met slot.assignedElements of ... werken